x-pack/solutions/security/plugins/security_solution/public/management/cypress/e2e/artifacts/blocklist_rbac.cy.ts

```diff
@@ -8,7 +8,9 @@
 import { getArtifactsListTestDataForArtifact } from '../../fixtures/artifacts_page';
 import { getArtifactMockedDataTests } from '../../support/artifacts_rbac_runner';
 
-describe(
+// Tests are not stable following the enablement of feature flag for space awareness. Issue is
+// being worked and these will be re-enabled soon.
+describe.skip(
```

Comment:
fyi, there should be no need to skip these tests, as the bugfix allowing this test to pass has been merged to main:
- https://github.com/elastic/kibana/pull/225678

(also, i'm reenabling these tests in 9.1 here: https://github.com/elastic/kibana/pull/228030)

Reply:
Thanks @gergoabraham . I will pull `upstream/main` back into this PR.